test/data/scheme-id.js

```diff
@@ -0,0 +1,14 @@
+const SCHEMES = [{ schemeId: 1, name: 'SFI' }, { schemeId: 2, name: 'SFI Pilot' }]
```

Comment:
Inclusion of a testing data module where other tests can require this for consistency -- is this a correct strat?